tpmutil/run_emu.go

```diff
@@ -0,0 +1,103 @@
+//go:build !windows && emu
```

Comment:
generally this kind of tag is not put before copyright, please see other u-root files.